scrapy/item.py

```diff
@@ -4,16 +4,22 @@
 See documentation in docs/topics/item.rst
 """
 
+from abc import ABCMeta
 from pprint import pformat
-from collections import MutableMapping
 from copy import deepcopy
+import collections
 
-from abc import ABCMeta
 import six
 
 from scrapy.utils.trackref import object_ref
 
 
+if six.PY3:
```

Comment:
It is better to use six.PY2 instead of six.PY3, because the condition may stop working (or stop making sense) when Python 4 is released